lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -6,10 +6,16 @@
                 .replace(/>/g, '&gt;')
         }
     %>
-    <div class='exam-text'>
+    <div class='exam-text js-exam-text' data-show-long="true">
         <% // xss-lint: disable=underscore-not-escaped %>
-        <%= interpolate_text('You are taking "{exam_link}" as a {exam_type} exam. The timer on the right shows the time remaining in the exam.', {exam_link: "<a href='" + exam_url_path + "'>"+gtLtEscape(exam_display_name)+"</a>", exam_type: (!_.isUndefined(arguments[0].exam_type)) ? exam_type : gettext('timed')}) %>
-        <%- gettext('To receive credit for problems, you must select "Submit" for each problem before you select "End My Exam".') %>
+        <%= interpolate_text('You are taking "{exam_link}" as a {exam_type} exam. ', {exam_link: "<a href='" + exam_url_path + "'>"+gtLtEscape(exam_display_name)+"</a>", exam_type: (!_.isUndefined(arguments[0].exam_type)) ? exam_type : gettext('timed')}) %>
+        <span class="js-exam-additional-text" aria-hidden="false">
+            <%- gettext('The timer on the right shows the time remaining in the exam.') %>
+            <%- gettext('To receive credit for problems, you must select "Submit" for each problem before you select "End My Exam".') %>
+        </span>
+        <button class="js-toggle-show-more toggle-show-more btn btn-pl-link btn-link" data-show-more-text="<%- gettext('Show More') %>" data-show-less-text="<%- gettext('Show Less') %>">
```

Comment:
I'm assuming `btn-pl-link` refers to a pattern library class, but I don't see it in the pattern library. Is there a reason you added it?

Reply:
Force of habit, since some other classes did need to be applied both in pl and post-pl versions to look good in both places. I'll remove it.

Where did you check for this?

Reply:
I checked in edx-platform, edx-proctoring, [edx-bootstrap](https://github.com/edx/edx-bootstrap) (didn't expect it ot be there, but checked just in case), and [ux-pattern-library](https://github.com/edx/ux-pattern-library).